src/scripts/ui/PlayerTradeComponent.tsx

```diff
@@ -60,7 +163,23 @@ export function PlayerTradeComponent({ gameState, xy }: IBuildingComponentProps)
                      <th>{t(L.PlayerTradeFrom)}</th>
                      <th></th>
                   </tr>
-                  {trades.map((trade) => {
+                  {trades
+                  .filter((t) => {
+                     if ( wantFilter === "" && offerFilter === "" ) {
```

Comment:
This file probably needs to be reformatted.

Reply:
File has been formatted.